omnisafe/evaluator.py

```diff
@@ -366,9 +386,14 @@ def evaluate(
                             'The policy must be provided or created before evaluating the agent.',
                         )
                 obs, rew, cost, terminated, truncated, _ = self._env.step(act)
+                if 'Saute' in self._cfgs['algo'] or 'Simmer' in self._cfgs['algo']:
+                    self._safety_obs -= cost.unsqueeze(-1) / self._safety_budget
+                    self._safety_obs /= self._cfgs.algo_cfgs.saute_gamma
 
                 ep_ret += rew.item()
                 ep_cost += (cost_criteria**length) * cost.item()
+                if 'EarlyTerminated' in self._cfgs['algo'] and ep_cost >= 25.0:
```

Comment:
```suggestion
                if 'EarlyTerminated' in self._cfgs['algo'] and ep_cost >= `25.0`:
```
Will we get it from `cfg` be better?

Reply:
Actually this is a python file, I don't think `25.0` makes sense.

Reply:
The `` just to highlight the code I am talking about. I mean we get this value from configs will be better.

Reply:
Got it. I have fixed them now.